pkg/filters/custom_filter.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package filters
+
+import (
+	"encoding/binary"
+	"math"
+	"math/big"
+)
+
+// CustomFilter is a custom filter
+type CustomFilter struct {
+	seed              int
+	k                 int
```

Comment:
I would hardcode `k=4` because it's good enough for our FP rate. Mainly for performance.